plugins/app/src/main/AndroidManifest.xml

```diff
@@ -27,6 +29,30 @@
                 android:name="android.support.PARENT_ACTIVITY"
                 android:value=".activity.FeatureOverviewActivity"/>
         </activity>
+
+        <activity
+            android:name=".activity.location.MyLocationModesActivity"
```

Comment:
We used the My prefix for API compability, with this plugin this isn't a requirement anymore. I would suggest naming all things just Location instead of MyLocation.